### ISSUE TITLE
Picklist size issue

### ISSUE THREAD

**Issue by ladyhang** [jangidyogi94/SFDC-Recipes#2]
Anyone else encountering picklist values are getting cut off when being displayed?

**Comment by padma-int** [comment]
yes , it is giving cut off , when we are few records and editing and clicking pick list.
